source/includes/_dataset_resources.md

```diff
@@ -209,6 +209,36 @@ The payload must be a valid resource object without any uid.
 
 The newly updated [resource object](#the-resource-object).
 
+## Download a resource
+
+This API endpoint is meant to download a resource specified with its UID.
```

Comment:
> download a resource specified with its UID.

And below:

> Returns: The resource with the specified UID

To me, in the context of this documentation, a resource is the Opendatasoft object linked to a dataset and this endpoint is to download the file linked to this resource.

Personally I find it more clear to say "the file used by the resource" or something like that, instead of "the resource" , since this route doesn't return the "opendatasoft resource object linked to a dataset", but maybe it's just me

Reply:
Got it, I'll change the wording a suggested (i.e. "the file used by the resource").